### ISSUE TITLE
train data

### ISSUE THREAD

**Issue by Sunnycl** [grekowj/musgenvaecnn_4v#1]
HI，when i ran main.py , i got 
`No such file or directory ./training_data/my_tensor_338_v2.npz`
how can i get it

**Comment by grekowj** [comment]
Hello,   I just added this file  ./training_data/my_tensor_338_v2.npz   Best  wishes ,  Jacek  Dnia 06 września 2023 10:09 shelly ***@***.***&gt; napisał(a):    HI，when i ran main.py , i got  No such file or directory ./training_data/my_tensor_338_v2.npz  how can i get it    —  Reply to this email directly,  github.com view it on GitHub , or  github.com unsubscribe .  You are receiving this because you are subscribed to this thread. Message ID:  &lt;grekowj/musgenvaecnn_4v/issues/1 @ github . com&gt;

**Comment by Sunnycl** [comment]
> Hello, I just added this file ./training_data/my_tensor_338_v2.npz Best wishes , Jacek Dnia 06 września 2023 10:09 shelly ***@***.***&gt; napisał(a): HI，when i ran main.py , i got No such file or directory ./training_data/my_tensor_338_v2.npz how can i get it — Reply to this email directly, github.com view it on GitHub , or github.com unsubscribe . You are receiving this because you are subscribed to this thread. Message ID: &lt;grekowj/musgenvaecnn_4v/issues/1 @ github . com&gt;

thank you so much, I still have a doubt. In the paper, you used Latent loss (LL). Does this mean that the latent space obeys Gaussian distribution with mean mu and variance δ??

**Comment by grekowj** [comment]
Hello,  Latent loss (kl_loss in the code) is calculated using the Kullback-Leibler divergence, which calculates  the distance between the the Gaussian distribution and the actual distribution  in latent vector z  At the end of VAE training, the latent space should have a Gaussian distribution. ( www.jeremyjordan.me https://www.jeremyjordan.me/variational-autoencoders/ )   In Code:  # ********************************************************************          reconstruction_loss = tf.reduce_mean(                      tf.reduce_sum(                          tf.losses.mean_squared_error(data, reconstruction))                    )     kl_loss = -0.5 * (1 + z_log_var - tf.square(z_mean) - tf.exp(z_log_var))      kl_loss = tf.reduce_mean(tf.reduce_sum(kl_loss, axis=1))                       total_loss = reconstruction_loss + Beta_KL * kl_loss   # ********************************************************************   Best wishes,   JAcek          Dnia 20 września 2023 12:09 shelly ***@***.***&gt; napisał(a):   Hello, I just added this file ./training_data/my_tensor_338_v2.npz Best wishes , Jacek Dnia 06 września 2023 10:09 shelly  @ .***&gt; napisał(a): HI，when i ran main.py , i got No such file or directory ./training_data/my_tensor_338_v2.npz how can i get it — Reply to this email directly, github.com view it on GitHub , or github.com unsubscribe . You are receiving this because you are subscribed to this thread. Message ID: &lt; github.com /issues/1  @ github . com&gt;  thank you so much, I still have a doubt. In the paper, you used Latent loss (LL). Does this mean that the latent space obeys Gaussian distribution with mean mu and variance δ??   —  Reply to this email directly,  github.com view it on GitHub , or  github.com unsubscribe .  You are receiving this because you commented. Message ID:  &lt;grekowj/musgenvaecnn_4v/issues/1/1727449540 @ github . com&gt;
